src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -45,7 +40,6 @@ public class EditCommand extends Command {
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_TAG + "TAG] "
             + "[" + PREFIX_GENDER + "GENDER]"
             + "[" + PREFIX_DOB + "DATE OF BIRTH] ...\n"
```

Comment:
Same reasoning as before!
```suggestion
            + "[" + PREFIX_DOB + "DATE OF BIRTH] \n"
```